src/Scaffold_Command.php

```diff
@@ -902,15 +902,18 @@ private function pluralize( $word ) {
 			'/$/'                      => 's'
 		);
 
-		$uncountable = array( 'equipment', 'information', 'rice', 'money', 'species', 'series', 'fish', 'sheep' );
+		$uncountable = array( 'sheep', 'fish', 'deer', 'series', 'species', 'money', 'rice', 'information', 'equipment' );
```

Comment:
@miya0001 
$uncountable = array( 'sheep', 'fish', 'deer', 'series', 'species', 'money', 'rice', 'information', 'equipment' );
do you want this form 
$uncountable = array(   'deer','equipment', 'fish' , 'information',  'money', 'rice', 'series', 'sheep', 'species' );